docs/release_notes.rst

```diff
@@ -1,6 +1,40 @@
 Release Notes
 =============
 
+v3.0.0
+-----
+
+:date: 2017-06-09
+
+This is a major release with breaking changes.
+
+``MapAttribute`` now allows pythonic access when recursively defined.
+If you were not using the ``attr_name=`` kwarg then you should have no problems upgrading.
+Previously defined non subclassed ``MapAttributes`` (raw ``MapAttributes``) that were members of a subclassed ``MapAttribute`` (typed ``MapAttributes``) would have to be accessed like a dictionary.
+Now object access is possible and recommended. See [here](https://github.com/pynamodb/PynamoDB/blob/master/pynamodb/tests/test_attributes.py#L671) for a test example.
+Access via the ``attr_name``, also known as the DynamoDB name, will now throw an ``AttributeError``.
+
+``UnicodeSetAttributes`` do not json serialize or deserialize anymore.
+We deprecated the functionality of json serializing as of ``1.6.0`` but left the deserialization functionality in there so people could migrate away from the old functionality. 
+If you have any ``UnicodeSetAttributes`` persisted in dynamo you will need to migrate your data or manage the json encoding and decoding in application if you were relying on that functionality.
```

Comment:
technically it's if they have any ``UnicodeSetAttributes`` that have _not_ been written with a pynamodb version >= 1.6.0. I'd also say "manage json encoding/decoding with a custom attribute in application"